mkdocs.yml

```diff
@@ -0,0 +1,82 @@
+site_name: Claudie Docs
+site_url: https://docs.claudie.io/
+repo_url: https://github.com/berops/claudie
+repo_name: claudie
+
+markdown_extensions:
+  - admonition
+  - pymdownx.details
+  - pymdownx.superfences
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
+  - pymdownx.tasklist:
+      custom_checkbox: true
+  - toc:
+      permalink: true
+
+plugins:
+  - mike:
+      canonical_version: null
+      version_selector: true
+  - search:
+      lang: en
+  - glightbox:
+      width: 100%
+      height: auto
+      zoomable: true
+
+copyright: Copyright &copy; 2023 BerOps s.r.o.
+
+theme:
+  name: material
+  features:
+    - content.code.copy
+    - navigation.top
+  palette:
+    - primary: white
+    - accent: black
+  logo: hexagon_blue.png
```

Comment:
hasn't this picture already been present in the repo somewhere?
Do we need to upload it anew?

Reply:
I guess this is the [picture ](https://github.com/berops/claudie/blob/feature/claudie-docs/docs/logo%20claudie_blue_no_BG.svg) you have mentioned.
In my opinion, we need this one, because the favicon and the the logo on the docs.claudie.io won't look that good, when there is also a caption "claudie Multi-cloud Kubernetes"

Reply:
ok, thanks